src/datasets/search/client_ltv/reference.md

```diff
@@ -16,7 +16,7 @@ SELECT
   AVG(IF(pred_num_days_seeing_ads > 0, 1, 0)) as pct_predicted_ad_viewers_next_year,
   AVG(IF(pred_num_days_clicking_ads > 0, 1, 0)) as pct_predicted_ad_clickers_next_year,
 FROM
-  `moz-fx-data-shared-prod`.revenue_derived.client_ltv_v1
+  `moz-fx-data-shared-prod`.revenue.client_ltv
```

Comment:
This is fine but generally we should use views in `mozdata` instead of `moz-fx-data-shared-prod` in user-facing documentation.